### PR TITLE
feat(dashboard): E2E tests implementation on dashboard permission management

### PR DESCRIPTION
**Pull request by paul-oureib**
## Description

Implementing the E2E tests from "Dashboards - Access rights management - Managing permissions of a dashboard"
(Also adding titles on all dashboard .feature tickets since I forgot to do it on all test implementations so far, and generally improved the access rights management feature descriptions)

**Fixes** # MON-20658

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Feature: As a Centreon User with dashboard administration rights,
```
Comment (bdauria) [inline centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature:1]: This should have a name

Comment (paul-oureib) [reply centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature:1]: what do you mean?

Comment (bdauria) [reply centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature:1]: You wrote a description but forgot to give your feature a name

Comment (paul-oureib) [reply centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature:1]: yeah correct
i think that's something i forgot to add on all dashboard test implementations 🫠 so i might add titles on all of them in this PR, will do that now

Comment (paul-oureib) [reply centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature]: done 👍

File: centreon/tests/e2e/cypress/e2e/Dashboards/05-dashboard-check-permissions.feature [hunk 2]
```diff
@@ -1,6 +1,7 @@
-Feature: As a Centreon User with dashboard administration rights,
+Feature: Check dashboard permissions
```
Comment (bdauria) [inline centreon/tests/e2e/cypress/e2e/Dashboards/05-dashboard-check-permissions.feature:1]: Check vs checking?

File: centreon/tests/e2e/cypress/e2e/Dashboards/06-dashboard-sharing.feature [hunk 3]
```diff
@@ -1,6 +1,7 @@
-Feature: As a Centreon User with dashboard administration rights,
-  I need to list all dashboards and user/user group rights to each, and update access rights if needed
-  so I can provision accesses or re-direct ownership if a user is not supposed to use Centreon anymore
+Feature: Share a dashboard
```
Comment (bdauria) [inline centreon/tests/e2e/cypress/e2e/Dashboards/06-dashboard-sharing.feature:1]:
```suggestion
Feature: Sharing a dashboard
```

**Review by Thebarda — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**